packages/launcher/src/main/rpc/app.js

```diff
@@ -31,9 +31,7 @@ class CommsSubscription extends ContextSubscription<RxSubscription> {
   }
 }
 
-const getStorageManifestHash = async (
-  ctx: AppContext,
-): Promise<Array<string>> => {
+const getStorageManifestHash = async (ctx: AppContext): Promise<Object> => {
```

Comment:
Returned type could be explicitly set to `Promise<{ feedHash: string, manifestHash: string }>` as the shape is known, it's more specific than `Object`.

Reply:
Nice, I didn't know that you could specify keys in the object for the type :+1: